exercises/concept/chess-game/.docs/instructions.md

```diff
@@ -0,0 +1,71 @@
+# Instructions
+
+As a chess enthusiast, you would like to write your own version of the game.
+Yes, there maybe plenty of implementations of chess available online already, but yours will be unique!
+
+You start with implementing a basic movement system for the pieces.
+
+The chess game will be played on a board that is 8 squares wide and 8 squares high.
```

Comment:
```suggestion
The chess game will be played on a board that is 8 squares wide and 8 squares long.
```

I think you want long here because you're talking about the length and width of the board, but "high" refers to the height.